tests/release/pipelines/fbc_release.go

```diff
@@ -207,8 +207,8 @@ func assertReleasePipelineRunSucceeded(devFw, managedFw framework.Framework, dev
 	mFw = releasecommon.NewFramework(managedWorkspace)
 	// Create a ticker that ticks every 3 minutes
 	ticker := time.NewTicker(3 * time.Minute)
-	// Schedule the stop of the ticker after 15 minutes
-	time.AfterFunc(15*time.Minute, func() {
+	// Schedule the stop of the ticker after 45 minutes
+	time.AfterFunc(45*time.Minute, func() {
```

Comment:
This is not related at all with this PR. It should be removed.

Reply:
It can help with issue #6 in [doc](https://docs.google.com/document/d/1jwdu89EVr7dV3oBCEyOn7DS_5KRkl3tfF-mIebVmOeA/edit).  For the other test cases, I'd like to increase them also to avoid error cause by token expiration.

Reply:
Updated